google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -39,6 +40,7 @@ private SessionPoolOptions(Builder builder) {
     this.writeSessionsFraction = builder.writeSessionsFraction;
     this.actionOnExhaustion = builder.actionOnExhaustion;
     this.actionOnSessionNotFound = builder.actionOnSessionNotFound;
+    this.waitForSessionTimeoutMillis = builder.blockForSessionTimeoutMillis;
```

Comment:
can we make these variables all 'wait...' or all 'block..'

Reply:
Changed to 'wait...'.